packages/transactions/src/builders.ts

```diff
@@ -593,12 +594,24 @@ function deriveNetwork(transaction: StacksTransaction) {
   }
 }
 
+/** @deprecated Not used internally */
 export interface MultiSigOptions {
   numSignatures: number;
   publicKeys: string[];
   signerKeys?: string[];
 }
 
+export interface UnsignedMultiSigOptions {
+  numSignatures: number;
+  publicKeys: string[];
+  /** @experimental Use newer non-sequential multi-sig hashmode for transaction. Future releases may make this the default. */
```

Comment:
I'm sure we will want this to be the default once 3.0 is released

Reply:
Yes, we'll do a second release but that will have to be a breaking change for these helper methods.